ciongke/cmd/hook/githubagent.go

```diff
@@ -86,18 +86,29 @@ func (ga *GitHubAgent) handlePullRequestEvent(pr github.PullRequestEvent) error
 	log.Printf("%s/%s#%d %s: %s", owner, name, pr.Number, pr.Action, pr.PullRequest.HTMLURL)
 
 	switch pr.Action {
-	case "opened", "reopened", "synchronize":
-		trusted, err := ga.trustedPullRequest(pr.PullRequest)
+	case "opened":
+		// When a PR is opened, if the author is in the org then build it.
```

Comment:
might be worth adding your PR comment here, noting why we special-case "opened".